pkgs/by-name/ca/candy-icons/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "candy-icons";
+  version = "unstable-2023-10-03";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "candy-icons";
+    rev = "bcf24f3308cc5e39f3e7d1bb43cb26f627f9ba8f";
+    hash = "sha256-NCXxH04GvrPE5mQO7JrU6SkjA+BaZokhyjKsm8m9p3g=";
+  };
+
```

```suggestion

  nativeBuildInputs = [
    gtk3
  ];

  dontDropIconThemeCache = true;

```
See https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-packaging